experimental/README.md

```diff
@@ -42,6 +42,13 @@ Unlike the regular Docker binary, the experimental channels is built and updated
 
     This command downloads a test image and runs it in a container.
 
+### Latest Binary
+
+In case you wish to download the binary directly, the latest experimental binary can be downloaded from :
+```
+https://experimental.docker.com/builds/Linux/x86_64/docker-latest
+```
+
```

Comment:
I think we can use the same text here as https://docs.docker.com/installation/binaries/#get-the-linux-binary, and perhaps the description of how to obtain the .md5/.sha256 hashes (https://docs.docker.com/installation/binaries/#get-the-docker-binary)

Also a link to that document for further instructions could be helpful